thesis-de/content-de.tex

```diff
@@ -1888,16 +1888,104 @@ \section{Userstories}
 Tickets arbeiten können.
 \tm%
 
-\subsection{Userstory 1}
+\subsection{SSD Userstory}
+\\textbf{Name der Story:}\\ Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
```

Comment:
userstories werden immer aus der anwendersicht beschrieben:
http://www.ksimons.de/2015/03/user-stories-verstaendlich-erklaert/

Das englische equivalent hab ich hier im titel https://jira.virtapi.org/browse/DCC-45:
As a system adminsitrator I would like to know if I kill my SSDs
das sollte auch hier genutzt werden, nur auf deutsch.